handbook/ce/customer-support-onboarding.md

```diff
@@ -1,5 +1,204 @@
+* Learn about who our customers are by checking out the [personas](https://about.sourcegraph.com/handbook/marketing/personas) overview and how Sourcegraph is useful for them by reading a few [case studies](https://about.sourcegraph.com/case-studies).
+* Explore the [product overview](https://about.sourcegraph.com/) and for each feature, explore the related [documentation](https://docs.sourcegraph.com/getting-started).
+* Spend some time getting familiar with [the Sourcegraph glossary](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/enterprise/docs/glossary.md)
+* Read the last few release blog posts on [our blog](https://about.sourcegraph.com/blog) and decide how you want to remember where to find this to look at later and often (when a customer has an issue, it’s possible it could be caused by something in a recent release)
+* Get a Sourcegraph instance [deployed](https://docs.sourcegraph.com/admin/install) using the `docker-compose` flow. This will be your own test environment. Watch [Dave’s Kubernetes Walkthrough video](https://drive.google.com/drive/folders/1JqQlrmGX8u_EuhNptj-qrzuZ4RifhTuF) for an understanding of that setup process. If you get stuck, just post in our #customer-support-chat Slack channel to start and if we get stuck as a team, we will see if our CE counterparts can help by asking them in #ce-chat.
+* Set up your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/main/doc/dev/getting-started/index.md). If you get stuck, just post in our #customer-support-chat Slack channel to start and if we get stuck as a team, we will get help in #any-questions. You will often want to run Sourcegraph locally when you are troubleshooting.
+* Add Sourcegraph as [a browser search engine](https://docs.sourcegraph.com/integration/browser_search_engine). To search our private code, log in to our internal dogfood instance (`k8s.sgdev.org`) and add another entry: `https://k8s.sgdev.org/search?q=%s`.
+* Install a text editor of your choice.
+ 
+* Start monitoring Zendesk for new support issues and try to own at least one by the end of the week. It is okay if we miss SLA for these first few cases. Take your time trying to figure out the right initial response, following our [workflow](https://about.sourcegraph.com/handbook/ce/support-workflow) steps. This will feel painful and tedious for a bit -- this is a reflection of our team being so nascent and the opportunity we have to make this better (not of your ability). Work as a team to solve issues, run responses by each other, etc. The more activity we have in our #customer-support-chat Slack channel, the better. And, if you have not used Zendesk before and would like a crash course, just let the team know in our #customer-support-chat Slack channel and we will make it happen!
```

Comment:
You may want to make that workflow link relative! So just `./support-workflow.md`, I think.